docs/2_4_common_co-simulation.adoc

```diff
@@ -196,32 +196,25 @@ The Co-Simulation master signals to the FMU which features shall be used for the
 include::../headers/fmi3FunctionTypes.h[tag=Instantiate]
 ----
 
-Additionally the Co-Simulation master provides the required Co-Simulation features via the argument `fmuCoSimulationConfiguration`:
-
-[source, C]
-----
-include::../headers/fmi3FunctionTypes.h[tag=CoSimulationConfiguration]
-----
-
-The master can only select a certain Co-Simulation interface for an FMU if the following attributes are set in the  `modelDescription.xml`.
+The master can only select a certain Co-Simulation interface for an FMU if the following attributes are set in the `modelDescription.xml`.
 
 [cols="1,2",options="header"]
 |===
 |Co-Simulation interface
 |Required attributes
 
-|`fmi3CoSimulation`
+|Basic Co-Simulation
 |`canNotUseBasicCoSimulation = false`
 
-|`fmi3HybridCoSimulation`
+|Hybrid Co-Simulation
 |`providesHybridCoSimulation = true`, `canReturnEarlyAfterIntermediateUpdate = true`
 
-|`fmi3ScheduledCoSimulation`
+|fmi3ScheduledCoSimulation
```

Comment:
fmi3ScheduledCoSimulation --> Scheduled Co-Simulation

Reply:
This is now fixed with providesScheduledCoSimulation = true